Sprint-3/implement/get-angle-type.test.js

```diff
@@ -0,0 +1,88 @@
+// Implement a function getAngleType
+
+// Acceptance criteria:
+
+// Given an angle in degrees,
+// When the function getAngleType is called with this angle,
+// Then it should:
+
+// Identify Right Angles:
+// When the angle is exactly 90 degrees,
+// Then the function should return "Right angle"
+
+// Identify Acute Angles:
+// When the angle is less than 90 degrees,
+// Then the function should return "Acute angle"
+
+// Identify Obtuse Angles:
+// When the angle is greater than 90 degrees and less than 180 degrees,
+// Then the function should return "Obtuse angle"fed
+
+// Identify Straight Angles:
+// When the angle is exactly 180 degrees,
+// Then the function should return "Straight angle"
+
+// Identify Reflex Angles:
+// When the angle is greater than 180 degrees and less than 360 degrees,
+// Then the function should return "Reflex angle"
+
+function getAngleType(angle) {
+    if (angle === 90) {
+        return "Right angle";
+    } else if (angle < 90) {
+        return "Acute angle";
+    } else if (angle > 90 && angle < 180) {
+        return "Obtuse angle";
+    } else if (angle === 180) {
+        return "Straight angle";
+    } else if (angle > 180 && angle < 360) {
+        return "Reflex angle";
+    }
+    return "Invalid angle";
+}
+
+// console.assert tests
+// Test 1
+const currentOutput1 = getAngleType(90);
+const targetOutput1 = "Right angle";
+console.log(currentOutput1);
+console.assert(currentOutput1 === targetOutput1, `${currentOutput1} is not equal ${targetOutput1}`);
+
+// Test 2
```

Comment:
Instead of writing Test 2, what else could you say about this test?

Reply:
I can add to the description "Test to check if the input angle is an acute angle".